modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastsViewModel.kt

```diff
@@ -314,8 +314,9 @@ class PodcastsViewModel
     fun refreshSuggestedFolders() {
         viewModelScope.launch {
             if (FeatureFlag.isEnabled(Feature.SUGGESTED_FOLDERS)) {
-                val uuids = podcastManager.findSubscribedUuids()
-                suggestedFoldersManager.refreshSuggestedFolders(uuids)
+                podcastManager.findSubscribedFlow().collect { podcasts ->
+                    suggestedFoldersManager.refreshSuggestedFolders(podcasts.map { it.uuid })
+                }
```

Comment:
When signing into my account, I noticed that it keeps calling the server. 
<img width="2103" alt="Screenshot 2025-02-21 at 5 13 50 pm" src="https://github.com/user-attachments/assets/62de277d-f255-442a-84ea-f23060e5d0c0" />

Reply:
I will investigate how can I fix the issue of missing podcasts in suggested folders without needing to monitor the subscribed changes

Reply:
So, I changed the approach here [37a522](https://github.com/Automattic/pocket-casts-android/pull/3630/commits/d37a522d6c04f73fd0ef8bdc7d154dc0b47e94dd)

Reply:
Thanks for changing that. Whenever I go back to the podcasts tab, it calls the endpoint even if I haven't added any podcasts or if I'm not tapping the suggested folder icon as a Plus user. I wonder if this is still too much? Especially if the server calls take a while to run.  

I initially thought we could store a setting for the latest podcast added date and only update it if the date changed, but that doesn't take into consideration when you unfollow a podcast. Is a hash of the subscribed UUIDs too much?

Reply:
I created this thread for us to talk about this p1740401875334509-slack-C08BRS7N9UL

Reply:
I've updated the code with the hash approach we talked about @geekygecko